discord/asset.py

```diff
@@ -326,6 +326,14 @@ def _from_user_banner(cls, state: _State, user_id: int, banner_hash: str) -> Sel
             animated=animated,
         )
 
+    @classmethod
+    def _from_soundboard_sound(cls, state: _State, sound_id: int) -> Self:
+        return cls(
+            state,
+            url=f'{cls.BASE}/soundboard-sounds/{sound_id}',
+            key=str(sound_id),
```

Comment:
If the key of the asset is the ID then it should probably be an asset mixin. Asset keys are more like hashes, at least so far.